src/moin/storage/middleware/indexing.py

```diff
@@ -328,7 +334,10 @@ def __init__(self, index_storage, backend, wiki_name=None, acl_rights_contents=[
             # tokenized CONTENTTYPE from metadata
             CONTENTTYPE: TEXT(stored=True, multitoken_query="and", analyzer=MimeTokenizer()),
             # unmodified list of TAGS from metadata
-            TAGS: ID(stored=True),
+            TAGS: KEYWORD(stored=True, commas=True, scorable=True, field_boost=2.0),
+            # search on HAS_TAGS improves response time of global tags
+            # https://whoosh.readthedocs.io/en/latest/api/query.html?highlight=#whoosh.query.Every
+            HAS_TAG: BOOLEAN(stored=False),
```

Comment:
ok, I've read that. ok, so HAS_TAG seems to be best practice for whoosh.

still not pretty imho having to do that, but assuming the docs are reflecting the current state of best practice, we can do it.